docs/modules/layers.rst

```diff
@@ -127,6 +127,25 @@ Layer classes: embedding layers
 -------------------------------
 
 .. autoclass:: EmbeddingLayer
```

Comment:
It is, sort of... you're adding a lot of lines in between `.. autoclass:: EmbeddingLayer` and the original `:members:` line that was paired with it, but the first thing you add is an empty line. So practically, it's gone.